python/GafferUI/MultiLineTextWidget.py

```diff
@@ -222,7 +222,7 @@ def setRole( self, role ) :
 		if role == self.getRole() :
 			return
 
-		self._qtWidget().setProperty( "gafferRole", GafferUI._Variant.toVariant( str( role ) ) )
```

Comment:
I'm going to stop commenting these now - probably easier for you to just grep the diff.

Reply:
Updated https://github.com/GafferHQ/gaffer/pull/3229/commits/960eac30e90e13188bdf5a114225b02f6b346e5c to leave it all alone. As agreed offline - we'll come back to unifying the use of variant or not another time.